contracts/RevenueDistributionToken.sol

```diff
@@ -75,7 +75,7 @@ contract RevenueDistributionToken is IRevenueDistributionToken, ERC20 {
         emit PendingOwnerSet(msg.sender, pendingOwner_);
     }
 
-    function updateVestingSchedule(uint256 vestingPeriod_) external override returns (uint256 issuanceRate_, uint256 freeAssets_) {
+    function updateVestingSchedule(uint256 vestingPeriod_) external virtual override returns (uint256 issuanceRate_, uint256 freeAssets_) {
```

Comment:
Can we also add `virtual` to ownership functions? In case we need to introduce a timelock or something like that.

Reply:
Yeah lets